kolibri/core/tasks/validation.py

```diff
@@ -15,6 +17,7 @@ class EnqueueArgsSerializer(serializers.Serializer):
     retry_interval = serializers.IntegerField(
         required=False, allow_null=True, min_value=0
     )
+    priority = serializers.IntegerField(required=False, allow_null=False)
```

Comment:
Could also use the `ChoiceField` here, with `Priority.Priorities` as the `choices` kwarg, which would mean that you wouldn't need the additional validation logic below: https://www.django-rest-framework.org/api-guide/fields/#choicefield